examples/3d/pcss.rs

```diff
@@ -0,0 +1,424 @@
+//! Demonstrates percentage-closer soft shadows (PCSS).
+
+use std::f32::consts::PI;
+
+use bevy::{
+    core_pipeline::{
+        experimental::taa::{TemporalAntiAliasPlugin, TemporalAntiAliasSettings},
+        prepass::{DepthPrepass, MotionVectorPrepass},
+        Skybox,
+    },
+    math::vec3,
+    pbr::{CubemapVisibleEntities, ShadowFilteringMethod, VisibleMeshEntities},
+    prelude::*,
+    render::{
+        camera::TemporalJitter,
+        primitives::{CubemapFrusta, Frustum},
+    },
+};
+
+use crate::widgets::{RadioButton, RadioButtonText, WidgetClickEvent, WidgetClickSender};
+
+#[path = "../helpers/widgets.rs"]
+mod widgets;
+
+/// The size of the soft shadow penumbras when PCSS is enabled.
```

Comment:
The penumbra scales according to ratios of distances between the receiving surface, the occluder, the light source, and the light source's radius. As such, this seems incorrect and misleading?

Reply:
I renamed this to `LIGHT_RADIUS` and reworded the comment.